content/docs/command-reference/exp/rename.md

```diff
@@ -11,8 +11,8 @@ usage: dvc exp rename [-h] [-q | -v]
                       [experiment] [name]
 
 positional arguments:
-   experiment    Experiments to rename.
-   name          Name of new experiment.
+   experiment    Experiment to rename
+   name          Name of new experiment
```

Comment:
Sorry, just committed a suggestion to revert the change on this line, but was there a reason behind wanting to change it @mattseddon?

Reply:
the experiment is still the same, it just has a new name. It is not a new experiment